post-type/campaign_core.js

```diff
@@ -173,8 +173,13 @@ window.campaign_scripts = {
     }
     return select_html
 
-  }
-
+  },
+  days_for_locale: (localeName = 'en-US', weekday = 'long')=>{
+    let now = new Date()
+    const format = new Intl.DateTimeFormat(localeName, { weekday }).format;
+    return [...Array(7).keys()]
+    .map((day) => format(new Date().getTime() - ( now.getDay() - day  ) * 86400000 ));
```

Comment:
what does this magic number represent?
could we put it into a labelled constant?

Reply:
@squigglybob which magic number?

Reply:
86400000

Reply:
sorry should have said magic constant number

Reply:
Fixed